docs/supplement.qmd

```diff
@@ -14,192 +14,258 @@ output:
+\mathbb{E}(N \ |\ Y = y) = \sum_{n=y}^{\infty}n \frac{\binom{n}{y} p^y(1-p)^{n-y}}{\sum_{i=y}^\infty \binom{i}{y}p^y(1-p)^{i-y}}
 $$
 
-Which is equivalent to:
+This is equivalent to
 
 $$
-\mathbb{E}[N | X = x] =  \frac{\sum_{n=x}^{\infty}n \binom{n}{x} p^x(1-p)^{n-x}}{\sum_{i=x}^\infty \binom{i}{x}p^x(1-p)^{i-x}}
+\mathbb{E}(N \ | \ Y = y) =  \frac{\sum_{n=y}^{\infty}n \binom{n}{y} p^y(1-p)^{n-y}}{\sum_{i=y}^\infty \binom{i}{y}p^y(1-p)^{i-y}}.
 $$
 
-Both the numerator and the denominator are known convergent series, with solutions available in standard libraries like Mathematica. We then get:
+Both the numerator and the denominator are known convergent series, with solutions available in standard libraries like Mathematica @Mathematica2024. We then get
 
 $$
-\mathbb{E}[N | X = x]  = \frac{(x + 1 -p)/p^2}{1/p} = \frac{x + 1 - p}{p}
-$$ This formula allows us to have a defined $\mathbb{E}[N | X = x]$ even when $X = 0$.
+\mathbb{E}(N \ |\ Y = y)  = \frac{(y + 1 -p)/p^2}{1/p} = \frac{y + 1 - p}{p},
+$$ which corresponds to the corrected point estimate provided in equation @eq-correction.
 
 ### Description of KIT simple nowcast implementation issue and revised nowcasts
 
-When verifying the KIT simple nowcast implementation, we noticed that the code used to generate the KIT simple nowcasts in real-time contained an additional indexed element in the delay PMF for all counts observed beyond the maximum delay thus far. In the generation of retrospective nowcasts, these values were being handled as having been observed as 0s, when in fact they had yet to be observed in all of the retrospective nowcasts. This resulted in a comparison of a 0 to a later observed value – which has the impact of inflating the dispersion estimates, and may explain why the KIT simple nowcast method in Wolffram et al. [@Wolffram2023] overcovers relative to the other methods. In the latest implementation in the RESPINOW Hub [@respinow_hub_2025], the authors have removed the density beyond the maximum delay column from their implementation, and thus it no longer contains this issue.
+When verifying the KIT simple nowcast implementation, we noticed that the code used to generate the KIT simple nowcasts in real-time contained an additional indexed element in the delay PMF for all counts observed beyond the maximum delay thus far. In the generation of retrospective nowcasts, these values were being handled as having been observed as 0s, when in fact they had yet to be observed in all of the retrospective nowcasts. This resulted in a comparison of a 0 to a later observed value ??? which has the impact of inflating the dispersion estimates, and may explain why the KIT simple nowcast method in Wolffram et al. [@Wolffram2023] overcovers relative to the other methods. In the latest implementation in the RESPINOW Hub [@respinow_hub_2025], the authors have removed the density beyond the maximum delay column from their implementation, and thus it no longer contains this issue.
```

Comment:
Because we have a prediction of 0, and then we have a corresponding observation that doesn't have to be 0  -- we will sum them with the other correct predictions and observations but this should still in general bias towards larger error than if that last delay entry had been excluded.